docs/docs/external-plugins.md

```diff
@@ -241,15 +305,53 @@ bash-3.2$ echo "$?"
 0
 ```
 
+### Tuple Value
+
+`exec`'s value can only be used to specify using a built-in _transport_ provided by Wash. The method tuple must be
+```
+[
+  "exec",
+  {
+    "transport": <transport>,
+    "options": { ... }
+  }
+]
+```
+where the object in the `options` field specifies transport-specific options. Supported transports are:
+
+* `ssh`: connect to an SSH server. It will look up port, user, and other configuration by exact hostname match from default SSH config files (global SSH config is currently ignored). If present, a local SSH agent will be used for authentication. The known hosts file will be ignored if StrictHostKeyChecking=no in your SSH config. _Options_ (string values unless otherwise specified):
```

Comment:
I'd indicate somewhere that the `ssh` transport uses SSH to implement your Exec (unless that's what "connect to an SSH server" means).